test/LEGO.AsyncAPI.Tests/Bindings/CustomBinding_Should.cs

```diff
@@ -54,7 +54,7 @@ public class MyBinding : ChannelBinding<MyBinding>
             { "custom", (a, n) => { a.Custom = n.GetScalarValue(); } },
             { "any", (a, n) => { a.Any = n.CreateAny(); } },
             { "nestedConfiguration", (a, n) => { a.NestedConfiguration = n.ParseMapWithExtensions(NestedConfiguration.FixedFieldMap); } },
-            { "myList", (a, n) => { a.MyList = n.CreateList((x) => x.GetScalarValue()); } }
+            { "myList", (a, n) => { a.MyList = n.CreateSimpleList((x) => x.GetScalarValue()); } }
```

Comment:
Ignore me, I was using the wrong method in the FixedFieldMap so instead of using `CreateList()` I need to use `CreateSimpleList()`. @VisualBean

Reply:
Perhaps we should rename `CreateList` to `CreateMap`